core/src/banking_stage/consume_worker.rs

```diff
@@ -644,6 +649,12 @@ impl ConsumeWorkerTransactionErrorMetrics {
                     .swap(0, Ordering::Relaxed),
                 i64
             ),
+            (
+                "invalid_compute_budget",
+                self.invalid_compute_budget
+                    .swap(0, Ordering::Relaxed),
+                i64
+            ),
```

Comment:
Nope, only from banking stage. I don't think it makes too much sense to report stats for individual error types from replay because it would mean the block is invalid and that rarely if ever happens.